README.md

```diff
@@ -105,6 +105,23 @@ sudo dnf install libappindicator-gtk3-devel
 
 Fedora does not have the Ayatana package yet, so you need to use the GTK one, see the [feature flags documentation](https://docs.rs/wry/latest/wry/#feature-flags).
 
+#### NixOS
+Latest Packages for Development:
+```nix
+let
+  # Unstable Channel | Rolling Release
+  pkgs = import (fetchTarball("channel:nixpkgs-unstable")) { };
+
+  packages = with pkgs; [
+    pkg-config
+    webkitgtk_4_1
```

Comment:
does nix have `libappindicator` or `libayatana` pkgs? if so, we should add them here with a note like other OSes